### PR TITLE
feat: add futures/option CNN_Cnn_Fear_Greed_Index

### PR DESCRIPTION
**Pull request by richardkuo1999**

### CODE REVIEW

**Comment by richardkuo1999** [comment]
@machineCYC @linsamtw 
Please review it, thank you.

File: FinMind/data/data_loader.py [hunk 1]
```diff
@@ -1956,6 +1956,31 @@ def taiwan_stock_trading_daily_report_secid_agg(
         )
         return stock_trading_daily_report_secid_agg
 
+    def Cnn_Fear_Greed_Index(
```
Comment (linsamtw) [inline FinMind/data/data_loader.py:1959]: 幫我用小寫
cnn_fear_greed_index

**Comment by linsamtw** [comment]
@richardkuo1999 好的，我們這周會進行 review

File: FinMind/data/data_loader.py (hunk 1 above)
Comment (linsamtw) [inline FinMind/data/data_loader.py:1959]: 幫我改成小寫，cnn_fear_greed_index

**Pull request merged**